<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rem</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>801</width>
      <height>701</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>341</width>
       <height>671</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/newPrefix/glass.jpg);
border-top-left-radius: 50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>160</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox#comboBox{
	background-color:rgba(0, 0, 0, 0);
	font: 14pt &quot;Segoe UI&quot;;
	color:rgba(0, 0, 0, 0);
	border: none;
	border-bottom: 2px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0.970149, y2:0.983, stop:0 rgba(170, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}

</string>
     </property>
     <item>
      <property name="text">
       <string>Mai Văn Anh</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Trần Toàn Thắng</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nguyễn Bá Quốc Tài</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rem</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="Password">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>230</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: rgba(0, 0, 42, 0);
	font: 14pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
	border: none;
	border-bottom: 2px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0.970149, y2:0.983, stop:0 rgba(170, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}

QLineEdit:hover{
	background-color: rgba(0, 0, 42, 160);
}

</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>Enter your Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>70</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>OCR A Extended</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255 , 255);</string>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EnterButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>233</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 80);
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="User">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>160</y>
       <width>335</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: rgba(0, 0, 42, 0);
	font: 14pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
	border: none;
	border-bottom: 2px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0.970149, y2:0.983, stop:0 rgba(170, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}

QLineEdit:hover{
	background-color: rgba(0, 0, 42, 160);
}

</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string comment="10">Enter your User</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>341</width>
       <height>671</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 80);
border-top-left-radius: 50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>411</width>
       <height>671</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0.191847 rgba(0, 55, 255, 255), stop:0.467626 rgba(0, 121, 255, 255), stop:0.977218 rgba(0, 255, 251, 255));
border-bottom-right-radius: 50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="close_button">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{
	background-color: rgb(255, 10, 10);
}

QPushButton{
	border: none;
	background-color: rgba(65, 68, 65, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../image/closebutton-removebg-preview.png</normaloff>../image/closebutton-removebg-preview.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>53</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="BoxButton">
     <property name="geometry">
      <rect>
       <x>721</x>
       <y>160</y>
       <width>21</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border : none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../image/pngtree-up-direction-arrow-icon-png-image_971517-removebg-preview (1).png</normaloff>../image/pngtree-up-direction-arrow-icon-png-image_971517-removebg-preview (1).png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="forgot_password">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>290</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{
	color: rgb(251, 255, 0);
}
QPushButton{
	color: rgb(255, 255, 255);
	border: none;
}</string>
     </property>
     <property name="text">
      <string>Forgot your password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>50</y>
       <width>211</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/newPrefix/278648865_716532036438113_3320425925480882445_n-removebg-preview.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Segoe UI&quot;;
text-decoration: underline;
color: rgb(99, 255, 3);</string>
     </property>
     <property name="text">
      <string>Hãy trồng rau</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Segoe UI&quot;;
text-decoration: underline;
color: rgb(99, 255, 3);</string>
     </property>
     <property name="text">
      <string>Nếu không GS Anh sẽ ăn thịt bạn</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Create_account">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>290</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{
	color: rgb(251, 255, 0);
}
QPushButton{
	color: rgb(255, 255, 255);
	border: none;
}</string>
     </property>
     <property name="text">
      <string>Create account</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>391</x>
       <y>330</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255, 0, 0, 0);</string>
     </property>
     <property name="text">
      <string>INcorrect password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eye">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>240</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{
	background-color: rgba(255, 255, 255, 0);
}

QPushButton:hover{
	background-color: rgba(0, 10, 10, 100);
}

QPushButton{
	border: none;
	background-color: rgba(65, 68, 65, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../image/eye-view-outline-icon-signs-symbols-can-be-used-web-logo-mobile-app-ui-ux-white-background-139746571-removebg-preview.png</normaloff>../image/eye-view-outline-icon-signs-symbols-can-be-used-web-logo-mobile-app-ui-ux-white-background-139746571-removebg-preview.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>29</width>
       <height>36</height>
      </size>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>-120</x>
       <y>440</y>
       <width>321</width>
       <height>291</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>340</y>
       <width>361</width>
       <height>341</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">color: qlineargradient(spread:reflect, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.124378 rgba(72, 72, 72, 243), stop:0.80597 rgba(255, 255, 255, 207));</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#f5f5f5;&quot;&gt;Design by: &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#f5f5f5;&quot;&gt;Tứ Trụ HCMUTE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;Mai Văn Anh 20139059&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;Đinh Thanh Tùng 20139096&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;Trần Toàn Thắng 20139015&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;Nguyễn Bá Quốc Tài 201390&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>label</zorder>
    <zorder>label_4</zorder>
    <zorder>Password</zorder>
    <zorder>EnterButton</zorder>
    <zorder>comboBox</zorder>
    <zorder>label_2</zorder>
    <zorder>User</zorder>
    <zorder>close_button</zorder>
    <zorder>BoxButton</zorder>
    <zorder>forgot_password</zorder>
    <zorder>label_3</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>Create_account</zorder>
    <zorder>label_5</zorder>
    <zorder>eye</zorder>
    <zorder>frame_2</zorder>
    <zorder>frame_3</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image_main.qrc"/>
 </resources>
 <connections/>
</ui>
